README.md

```diff
@@ -354,6 +354,34 @@ features (like associations, inheritance etc) are supported and covered. Other O
 issues as well, tests are in progress. All optional arguments (`camelize`, `include_root` etc) are also supported.
 Guides on where to use `#surrealize_collection` vs `#surrealize` for all ORMs are coming.
 
+### Root
+If you want to wrap the resulting JSON into a specified root key, you can pass optional `root` argument
+to `#surrealize` or `#build_schema`. The root key in this case will be taken from the class name of the
```

Comment:
> The root key in this case will be taken from the class name of the
surrealizable object.
This sentence is irrelevant here (it is for `include_root` argument)
Also please add link to here from table of contents.